doc/nrf/samples.rst

```diff
@@ -18,6 +18,15 @@ In addition, the |NCS| provides the following samples that showcase the use of a
    ../../samples/bluetooth/*/README
    ../../samples/bluetooth/mesh/*/README
 
+.. _edge_impulse_samples:
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Edge Impulse samples
+   :glob:
+
+   ../../samples/edge_impulse/*/README
```

Comment:
I am wondering why slashes are rendered differently in various places... see lines 18 and 19. Are these different characters?

Reply:
The `*text*` works as cursive in *.rst files, I guess that's the reason why it's rendered like that